.github/workflows/manifest-checks.yaml

```
@@ -34,6 +34,7 @@ jobs:
           tar zx -C /usr/local/bin
       - name: Run `kustomize build` across changed directories
         run: >-
+          set -e
```

This is already set on the shell (GHA defaults to `shell: /usr/bin/bash -e`) and doesn't catch the issue (docs: https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#The-Set-Builtin) 

> Exit immediately if a pipeline (see [Pipelines](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Pipelines)), which may consist of a single simple command (see [Simple Commands](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Simple-Commands)), a list (see [Lists of Commands](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Lists)), or a compound command (see [Compound Commands](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Compound-Commands)) returns a non-zero status.

The problem is the pipeline _is_ succeeding because it's not the last command that fails, we need to make it so that if _any_ part of the pipeline fails then the pipeline returns non-zero, i.e.:

```suggestion
          set -o pipefail
```

Also please add the same thing for the step that builds on `main`

---

[Applied](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/36/commits/7ead01b8855dc1ccd3446f316408297956a5f59b)